src/V3EmitCFunc.h

```diff
@@ -1089,7 +1141,9 @@ class EmitCFunc VL_NOT_FINAL : public EmitCConstInit {
         puts(")");
     }
     void visit(AstNewCopy* nodep) override {
-        puts("VL_NEW(" + prefixNameProtect(nodep->dtypep()) + ", ");
+        bool ctorNeedProc = constructorNeedsProcess(nodep->dtypep());
+        puts("VL_NEW(" + prefixNameProtect(nodep->dtypep())
+             + (ctorNeedProc ? ", vlProcess, " : ", "));
```

Comment:
It might be cleanest in all these to make a function that takes a bool and returns either "" or "vlProcess, ", then use that function everywhere you have the current conditionals.

Reply:
Done